assets/js/editor-components/search-list-control/types.ts

```diff
@@ -35,7 +35,7 @@ export interface ListItemsProps
 }
 
 export type SearchListItem = {
-	breadcrumbs: string[];
+	breadcrumbs?: string[];
```

Comment:
Based on code checks this may not be present.

Reply:
According to [this](https://github.com/woocommerce/woocommerce-blocks/blob/0ade96139c899d39d9bc56386a5eba7fbe88a008/assets/js/utils/attributes.ts#L63), this property will always exist.